conda_smithy/configure_feedstock.py

```diff
@@ -1655,15 +1661,15 @@ def commit_changes(forge_file_directory, commit, cs_ver, cfp_ver, cb_ver):
         )
         if has_staged_changes:
             if commit:
-                git_args = ["git", "commit", "-m", "MNT: {}".format(msg)]
+                git_args = ["git", "commit", "-m", msg]
                 if commit == "edit":
                     git_args += ["--edit", "--status", "--verbose"]
                 subprocess.check_call(git_args, cwd=forge_file_directory)
                 logger.info("")
             else:
                 logger.info(
                     "You can commit the changes with:\n\n"
-                    '    git commit -m "MNT: {}"\n'.format(msg)
+                    '    git commit -m "MNT: Re-rendered"\n'
```

Comment:
I copy-paste this commit message a lot. Can we use the old way here?

Reply:
Yeah, I find it useful to have the extra info there

Reply:
I'm curious why copy-paste is used as opposed to `conda smithy rerender -c [edit|auto]`?

Reply:
I like to inspect the changes before they are committed. Also I usually have other changes to stage

Reply:
I think we leave the human-readable one when it prints and make it 50/72 when we auto-commit.